src/AVFoundation/Enums.cs

```diff
@@ -158,8 +158,8 @@ public enum AVCaptureWhiteBalanceMode : nint {
 
 #if !MONOMAC || !XAMCORE_4_0
 	[Flags]
-	[NoTV, NoWatch]
-	[iOS (4,0)]
+	[NoTV]
+	[iOS (4,0), Watch (3,0)]
```

Comment:
I don't see it added in the headers, nor does it seems used elsewhere in your PR ???